front_end/webserver.py

```diff
@@ -126,6 +127,7 @@ def prepare(self):
             if user_id:
                 user_info_var.set(content.get_user_info(user_id.decode()))
                 user_is_administrator_var.set(content.is_administrator(user_id.decode()))
+                user_is_student_var.set(content.is_student(user_id.decode()))
```

Comment:
Please see my other comment about removing the is_student function from content.py. It might make sense to keep the user_is_student_var context variable though.